app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
 
     <!-- Accessibility descriptions -->
     <string name="desc_navigation_btn">Open Navigation Drawer</string>
     <string name="desc_settings_btn">Open Settings</string>
-    <string name="desc_saved_servers_tab">Saved Servers</string>
-    <string name="desc_discovered_servers_tab">Discovered Servers</string>
-    <string name="desc_add_new_server_btn">Add new Server</string>
-    <string name="desc_save_discovered_server_btn">Save this Server</string>
-    <string name="desc_discovery_btn">Start Server Discovery</string>
-    <string name="desc_discovery_running">Discovering Servers on you network</string>
-    <string name="desc_keyboard_btn">Open Keyboard</string>
-    <string name="desc_virtual_keys_btn">Show Virtual Keys</string>
-    <string name="desc_reset_zoom_btn">Reset Zoom to 100%</string>
+    <string name="desc_saved_servers_tab">Saved</string>
+    <string name="desc_discovered_servers_tab">Discovered</string>
+    <string name="desc_add_new_server_btn">Add new</string>
+    <string name="desc_save_discovered_server_btn">Save this</string>
+    <string name="desc_discovery_btn">Discover</string>
+    <string name="desc_discovery_running">Discovering running servers on you network…</string>
+    <string name="desc_keyboard_btn">Open keyboard</string>
+    <string name="desc_virtual_keys_btn">Show virtual keys</string>
+    <string name="desc_reset_zoom_btn">Reset Zoom</string>
     <string name="desc_retry_connection_btn">Retry Connection</string>
-    <string name="desc_back_btn">Go Back</string>
-    <string name="desc_clear_url_btn">Clear URL</string>
+    <string name="desc_back_btn">Back</string>
+    <string name="desc_clear_url_btn">Clear</string>
     <string name="desc_server_url">Enter address of VNC server</string>
-    <string name="desc_help_btn">Show Help</string>
-    <string name="desc_discovered_list">List of automatically discovered servers</string>
-    <string name="desc_saved_servers_list">List of saved Servers</string>
+    <string name="desc_help_btn">Help</string>
+    <string name="desc_discovered_list">List of discovered servers</string>
+    <string name="desc_saved_servers_list">List of saved servers</string>
 
     <!-- Preference -->
     <string name="pref_ui">Interface</string>
     <string name="pref_ui_summary">Look and feel</string>
     <string name="pref_viewer">Viewer</string>
-    <string name="pref_viewer_summary">Picture-in-Picture, Zoom</string>
+    <string name="pref_viewer_summary">Picture-in-picture, zoom</string>
     <string name="pref_input">Input</string>
-    <string name="pref_input_summary">Gestures, Mouse, Virtual keys</string>
+    <string name="pref_input_summary">Gestures, mouse, virtual keys</string>
     <string name="pref_servers">Servers</string>
-    <string name="pref_server_summary">Discovery, Credentials</string>
+    <string name="pref_server_summary">Discovery, credentials</string>
```

Comment:
I don't particularly like the word "credentials".
Maybe an alternative can be found? (More about this above.)